sec.club/src/App.js

```diff
@@ -15,12 +15,30 @@ function App() {
     }
   })
   return (
-    <ThemeProvider theme={theme}>
-      <div className="App">
-        <Header></Header>
-      </div>
-    </ThemeProvider>
+    <Router>
+      <ThemeProvider theme={theme}>
+        <div className="App">
+          <Header></Header>
+          <Switch>
+            <Route path="/slp">
+              <h1>Add SLP info page here</h1>
+            </Route>
+            <Route path="/about">
+              <h1>Add What is SEC? page here</h1>
+            </Route>
+            <Route path="/team">
+              <h1>Add team information page here</h1>
+            </Route>
+            <Route path="/contact">
+              <h1>Add contact page here</h1>
+            </Route>
+            <Route exact path="/">
```

Comment:
Why does this one have `exact` and none of the others?

Reply:
`exact` is there because without it, the Router will display whatever is in the `/` route along with any other route thats being rendered. Basically if you didn't put `exact` then it would look for the `/` character in every other route (which they all have) and it would render extra stuff. Read this for a better explanation: https://stackoverflow.com/questions/49162311/react-difference-between-route-exact-path-and-route-path